modules/pubmaticBidAdapter.js

```diff
@@ -71,25 +71,25 @@ function _parseAdSlot(bid) {
   bid.params.adSlot = _cleanSlot(bid.params.adSlot);
 
   var slot = bid.params.adSlot;
-  var splits = slot.split(':');
+  try{
+    var splits = slot.split('@');
 
-  slot = splits[0];
-  if (splits.length == 2) {
-    bid.params.adUnitIndex = splits[1];
-  }
-  splits = slot.split('@');
-  if (splits.length != 2) {
-    utils.logWarn('AdSlot Error: adSlot not in required format');
-    return;
+    slot = splits[0];
+    if (splits.length == 2) {
+      bid.params.adUnitIndex = splits[1].split(":")[1];
```

Comment:
Will break in case the split on colon doesn't produce two elements. I believe this is not a mandatory check.